toggleQuotes.js

```diff
@@ -1,7 +1,7 @@
-function handleClick() {
-  browser.toggleQuotesApi.toggleQuotes();
-};
+browser.messageDisplayAction.onClicked.addListener((tab, info) => {
+  browser.toggleQuotesApi.toggleQuotes(tab.windowId);
+});
 
-browser.browserAction.onClicked.addListener(handleClick);
-
-browser.toggleQuotesApi.loadButton();
+browser.messageDisplay.onMessageDisplayed.addListener((tab, message) => {
+  browser.toggleQuotesApi.loadButton(tab.windowId);
+});
```

Comment:
Making this multi window aware. Instead of loading the button code when the API starts, load it whenever a message is displayed.